### PR TITLE
Adds new brain damage line

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

Updates the trauma/brain damage file with a new line.

## Why It's Good For The Game

Tormenting janny's is funny

## Changelog

:cl:
code: dendy sisters.. it's joever
/:cl:

### CODE REVIEW

**Comment by TheVekter** [comment]
Too wordy.

**Comment by dendyd00m** [comment]
YOU CJAN'T DO THIS TO MEEEEEEEE

File: strings/traumas.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
         "its gonna be like 9/11 all over again but insitaead is gonna be a bitc",
         "lifelike texture ;_;",
         "lol2cat",
+        "hey zishy, pelase don't use OOC language in IC like this: "[Common] Zishy Phoebe says, "The ai and borgs mettacomming I think" it upsets the audience! stay in character, we're professionals! if you believe someone is breaking the rules, please ahelp it.",
```
Comment (JohnFulpWillard) [inline strings/traumas.json:43]: ?

Comment (MrStonedOne) [inline strings/traumas.json:43]:
```suggestion
        "hey zishy, pelase don't use OOC language in IC like this: \"[Common] Zishy Phoebe says, \"The ai and borgs mettacomming I think\" it upsets the audience! stay in character, we're professionals! if you believe someone is breaking the rules, please ahelp it.",
```

**Comment by ATH1909** [comment]
don't we already have a reference to this

or is this a new incident

**Comment by carshalash** [comment]
> don't we already have a reference to this
> 
> or is this a new incident

yeah, just teasing a candymin over an accident.

**Comment by Ryll-Ryll** [comment]
🎉🎉🎉

**Comment by TheVekter** [comment]
it's so joever
